### PR TITLE
Update travis to test against rails 5

### PR DESCRIPTION
**Pull request by pyromaniac**

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -1,4 +1,4 @@
-%w(3.2 4.0 4.1 4.2).each do |version|
+%w(3.2 4.2 5.0).each do |version|
```
Comment (krzysiek1507) [inline Appraisals]: `3.2`?

Comment (pyromaniac) [inline Appraisals]: well, yeah, we still support it

Comment (krzysiek1507) [inline Appraisals]: `3.2` gemfiles are missing from `gemfiles` part in `travis.yml`. Is this correct?

Comment (pyromaniac) [inline Appraisals]: https://github.com/toptal/chewy/pull/420/files#diff-354f30a63fb0907d4ad57269548329e3R28

They are in martig/include section.

**Pull request merged**